server/utils.go

```diff
@@ -24,3 +26,40 @@ func getGPRCVersion() (string, error) {
 	}
 	return out, nil
 }
+
+// the first return value is true if path is created successfully, and is false otherwise.
+func parseDNSOptions(servers, searches []string, path string) (bool, error) {
+	nServers := len(servers)
+	nSearches := len(searches)
+	if nServers == 0 && nSearches == 0 {
+		return false, nil
+	}
+
+	if nSearches > 6 {
```

Comment:
No, this is just to document it better so someone can know why we are using 6 here. Of course, even just a comment will do.